localenv/README.md

```diff
@@ -1,14 +1,84 @@
 # Local Playground
 
+We have created a suite of packages that, together, mock an account servicing entity that has deployed Rafiki, exposing an [SPSP](../docs/glossary.md#simple-payments-setup-protocol-spsp) endpoint, the [Open Payments](../docs/glossary.md#open-payments) APIs with its required [GNAP](../docs/glossary.md#grant-negotiation-authorization-protocol) auth endpoints to request grants, a STREAM endpoint for receiving Interledger packets, and a UI to view and manage the Rafiki instance. Additionally, we provide a simple request signing service that is used by Postman to generate request signatures required by the Open Payments APIs.
+
+These packages include:
+
+- `backend` (SPSP, Open Payments APIs, GraphQL Admin APIs, STREAM endpoint)
+- `auth` (GNAP auth server)
+- `mock-account-servicing-entity` (mocks an [Account Servicing Entity](../docs/glossary.md#account-servicing-entity))
+- `frontend` (Remix app to expose a UI for Rafiki Admin management via interaction with the `backend` Admin APIs)
+- `local-http-signatures` (request signature generation for Postman)
+
+These packages depend on the following databases:
+
+- TigerBeetle or Postgres (accounting)
+- Postgres (Open Payments resources, auth resources)
+- Redis (STREAM details)
+
+We provide containerized versions of our packages together with two pre-configured docker-compose files ([peer1](./cloud-nine-wallet/docker-compose.yml) and [peer2](./happy-life-bank/docker-compose.yml)) to start two Mock Account Servicing Entities with their respective Rafiki backend and auth servers. They automatically peer and 2 to 3 user accounts are created on both of them.
+
 This environment will set up an playground where you can use the Rafiki Admin APIs and the Open Payments APIs.
 
-Prerequisites:
+## Environment overview
+
+![Docker compose environment](./docs/assets/img/localenv-architecture.png)
```

Comment:
Maybe we can just upload the images to github directly, instead of housing them in the repo? I think the excalidraw files should live in the repo, but not sure about the pngs

Reply:
I don't have strong opinions. Either way works for me.

Reply:
In that case, I'll just keep it as it is, makes it easier to work with the images (especially previewing the docs locally)